CLAUDE.md

```
@@ -0,0 +1,576 @@
+  
+  ```shell
+  atmos command subcommand <args> [options]
+  ```
+  
+  ## Examples
+  
+  ```shell
+  atmos command subcommand example1
+  atmos command subcommand example2 --flag=value
+  ```
+  ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)